### ISSUE TITLE
Add a variable for TV Shows release year

### ISSUE THREAD

**Issue by TheFrenchGhosty** [jkwill87/mnamer#89]
According to https://github.com/jkwill87/mnamer/wiki/Formatting there is no way to add a TV Show release year in the ```episode-format``` argument.

From what I see on TVDB it is displayed on the search page, so I'm guessing it's also available via the API. It would really be nice if there was a way to add it automatically, right now I have to add it by hand.

Edit: After more digging, I noticed that mnamer actually use TVmaze as the default API for the TV Shows, and while it doesn't provide the year, TVDB provide it in the title, any reason why tvdb isn't the default?

**Comment by jkwill87** [comment]
> According to https://github.com/jkwill87/mnamer/wiki/Formatting there is no way to add a TV Show release year in the episode-format argument.

That's correct. Ideally we could include year when there is ambiguity between series as seems to be done on the TVDb website. I'm not sure if there is a uniform way to accomplish this across providers but its something I can look into.

> any reason why tvdb isn't the default?

- Its unreliable and often down.
- It requires an API key. One is included with mnamer but since its shared between users it can result in rate limiting.

**Comment by jkwill87** [comment]
So for unrelated reasons TVDb has been set as the default provider in [v2.4.1](https://github.com/jkwill87/mnamer/releases/tag/2.4.1) 😅

**Comment by allenbina** [comment]
I would also like to see this added.  Plex recommends using the year in the TV shows, and some TV shows from different years have the same name.  https://support.plex.tv/articles/naming-and-organizing-your-tv-show-files/

**Comment by Nideapum** [comment]
You can take the year from the first episode (S01E01) this is the release date.

**Comment by stassinari** [comment]
I am _so close_ to getting rid of FileBot, but the release year for TV series is the one thing that makes mnamer renaming not 100% the same as my old one.

I'd love to be able to contribute, but I'm not sure I'll be able to any time soon (plus it's been forever since I've done any python).

So I guess another +1 for this issue.

**Comment by mattwaler** [comment]
Hey everybody, I'm loving this application and have the same issue. I utilized Claude Code to create a PR to resolve this:
https://github.com/jkwill87/mnamer/pull/385
